testing/toilet/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: z3bra <willy at mailoo dot org>
+# Maintainer: Shiz <hi@shiz.me>
+pkgname=toilet
+pkgver=0.3
+pkgrel=0
+pkgdesc="A free replacement for the FIGlet utility"
+url="http://caca.zoy.org/wiki/toilet"
+arch="all"
+license="WTFPL"
+makedepends="libcaca-dev"
+subpackages="$pkgname-doc"
+source="https://distfiles.alpinelinux.org/distfiles/$pkgname-$pkgver.tar.gz"
```

Comment:
This is not a git or svn repo, and it won't guarantee a build working if the server is down while a snapshot is being made (initially, for instance).